src/MCPClient/lib/clientScripts/archivematicaCreateMETS2.py

```diff
@@ -800,8 +800,8 @@ def createFileSec(directoryPath, parentDiv, baseDirectoryPath, baseDirectoryName
                 label = "mets.xml-%s" % (GROUPID)
                 dspace_dmdsecs = createDSpaceDMDSec(label, itemdirectoryPath, directoryPathSTR)
                 if dspace_dmdsecs:
-                    dmdSecs.extend(dspace_dmdsecs.values())
-                    ids = ' '.join(dspace_dmdsecs.keys())
+                    dmdSecs.extend(list(dspace_dmdsecs.values()))
+                    ids = ' '.join(list(dspace_dmdsecs.keys()))
```

Comment:
Is it really necessary to convert these dict views to lists? In Python 3.4 you can extend a list with the output of .values() and also join that dict view on a string.